cmd/server/server.go

```diff
@@ -0,0 +1,52 @@
+// Package server contains the code which can be used to start opi-spdk-bridge
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022 Dell Inc, or its subsidiaries.
+// Copyright (C) 2023 Intel Corporation
+package server
+
+import (
+	"flag"
+	"fmt"
+	"log"
```

Comment:
Don't use "log" for logging. It's way too limited in practice.

A reusable package should use some abstract logging interface. Whoever composes some binary then gets to decide which logging implementation to use.

For now, I recomment go-logr for this. In some future Go release, slog can replace it.

Reply:
I think we can add this log rework proposal into the project Issues